### PR TITLE
boveri Pathology_detail node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -46,7 +46,6 @@ properties:
   additional_pathology_findings:
     $ref:
       - "_terms.yaml#/additional_pathology_findings/common"
```
Comment (wwysoc2) [inline gdcdictionary/schemas/pathology_detail.yaml:48]: We need the enums to be here @ganjigajanan

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 2]
```diff
@@ -46,7 +46,6 @@ properties:
   additional_pathology_findings:
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:46]: DICT-96 says there are enums added:
Pleurodesis, Talc
Pleurodesis, NOS

I don't see them in the changes made.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/pathology_detail.yaml:48]: added the enums

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 2 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/pathology_detail.yaml:46]: added

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**